### PR TITLE
Update mobile-security-and-authentication.md

### PR DESCRIPTION
**Pull request by msbemba**
Corrected the invalid link to https://support.microsoft.com/en-us/help/17240/windows-10-mobile-find-phone

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointServer/administration/mobile-security-and-authentication.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ The functionality this service can provide includes the following:
 - Wipe your mobile device data.
     
 > [!NOTE]
-> To learn more about the Find My Phone service for Windows Phone, see [Find a lost phone](https://go.microsoft.com/fwlink/?LinkId=253669). 
+> To learn more about the Find My Phone service for Windows Phone, see [Find a lost phone](https://support.microsoft.com/en-us/help/17240/windows-10-mobile-find-phone).
```
Comment (nenonix) [inline SharePoint/SharePointServer/administration/mobile-security-and-authentication.md]:
```suggestion
> To learn more about the Find My Phone service for Windows Phone, see [Find a lost phone](https://support.microsoft.com/help/17240/windows-10-mobile-find-phone). 
```

**Comment by scanum** [comment]
@tomresing Changes look correct. Could you please review and merge? Thanks!

**Comment by scanum** [comment]
@tomresing Any updates in merging this PR? Changes look correct. Thanks

**Pull request merged**